internal/client.go

```diff
@@ -467,6 +466,12 @@ type (
 		// grpc.WithUnaryInterceptor option can be added since grpc.WithUnaryInterceptor is prepended to chains set with
 		// grpc.WithChainUnaryInterceptor.
 		DialOptions []grpc.DialOption
+
+		// Hidden for use by client overloads.
+		disableEagerConnection bool
+
+		// If not present, created during service client creation.
```

Comment:
May you add more javadoc about what it actually defines, not just how it's initialized? The name is not descriptive enough for me.

Reply:
Sure. This name has already been in use, I just re-added it here and this is internal btw, so the customer doesn't see it. But I will add documentation on what it is.